src/Styles/index.css

```diff
@@ -23,31 +24,80 @@
     color: black;
 }
 
-.fade{
-    height: 749px;
-    background: linear-gradient(70deg, #7FFFD4, #9932CC);
-    z-index: 0;
-    position: relative;
+.background{
+    background: #F5F5F5;
+}
+
+.background32{
+    background: red;
+    width: 100%;
+    height: 810px;
+
 }
 
 .header-scrolled{
-    background:white;
+    background:#C0C0C0;
     width: 100%;
     height: 80px;
     position: fixed;
     color: black;
     z-index: 5;
 }
 
+.about{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background: white;
+    height: 700px;
+    width: 990px;
+}
+
 .ava{
-    height: 300px;
-    width: 300px;
+    height: 600px;
+    width: 450px;
 }
 
-.about{
+.wish{
+    width: 100%;
+    background: white;
+}
+
+.colon-wish{
+    display: grid;
+    grid-template-columns: repeat(2, 2fr);
+    gap: 50px;
+    padding: 30px;
+    padding-left: 220px;
+    padding-right: 220px;
```

Comment:
Use rem or em

Reply:
In general, I advise you to read about units of measurement. I also think that the paddings are too big and could be made simpler somehow